pkg/ksync/cli.go

```diff
@@ -79,10 +79,7 @@ func install(latestVersion, binDir, fileName string) error {
 func (kcli *CLI) Version() (string, error) {
 	args := []string{"version"}
 	kcli.Runner.SetArgs(args)
-	out, err := kcli.Runner.RunWithoutRetry()
```

Comment:
Why are we ignoring the error here?

Reply:
`ksync version` does both a client and server version check. In a case when no server is present a warning is returned. The version parsing is still possible with the warning included in the output. Unfortunately there is no extra flag to do a client only version check.